semantics/src/main/kotlin/com/strumenta/kolasu/semantics/symbol/description/SymbolDescription.kt

```diff
@@ -0,0 +1,39 @@
+package com.strumenta.kolasu.semantics.symbol.description
+
+data class SymbolDescription(
+    val identifier: String,
+    val types: List<String>,
+    val properties: Map<String, ValueDescription>
+)
+
+sealed interface ValueDescription {
+    val value: Any?
+}
+
+data class BooleanValueDescription(
+    override val value: Boolean? = null
+) : ValueDescription
+
+data class IntegerValueDescription(
+    override val value: Int? = null
+) : ValueDescription
+
+data class StringValueDescription(
+    override val value: String? = null
+) : ValueDescription
+
+data class ReferenceValueDescription(
+    override val value: SymbolDescription? = null
+) : ValueDescription
+
+data class ListValueDescription(
+    override val value: List<ValueDescription> = emptyList()
+) : ValueDescription
+
+data class SetValueDescription(
+    override val value: Set<ValueDescription> = emptySet()
+) : ValueDescription
+
+data class MapValueDescription(
```

Comment:
We do not use maps in Kolasu

Reply:
Set and Map value descriptions will be removed (see discussion above)